web/client/utils/LayersUtils.js

```diff
@@ -298,13 +302,18 @@ const LayersUtils = {
             ...assign({}, layer.params ? {params: layer.params} : {})
         };
     },
+    REG_GEOSERVER_RULE,
+    findGeoServerName,
     getCapabilitiesUrl: (layer) => {
+        const matchedGeoServerName = findGeoServerName({url: layer.url});
```

Comment:
please add a comment that explains it is an optimization for GeoServer and otherwize it fails into default capabilities url